build-system/tasks/coverage-map/index.js

```diff
@@ -98,8 +98,8 @@ async function autoScroll(page) {
  */
 async function htmlTransform() {
   const {
-    transform,
     // @ts-ignore
+    transform,
```

Comment:
If `new-server` hasn't been built this will result in a type error on line 103.
```
build-system/tasks/coverage-map/index.js:103:15 - error TS2307: Cannot find module '../../server/new-server/transforms/dist/transform' or its corresponding type declarations.
```

Reply:
Aha. Looks like we need to ignore both spots. I've restored the other `@ts-ignore`. (Not sure if there's a better way to do this.)

Reply:
The entire statement is now on one line, so the ignore directive applies to both the `require` and the destructure key.